types/oracledb/index.d.ts

```diff
@@ -4667,6 +4667,97 @@ declare namespace OracleDB {
     function startup(creds: DBCredentials, opts?: StartupOptions): Promise<void>;
     function startup(creds: DBCredentials, opts: StartupOptions, cb: (err: Error) => void): void;
     function startup(creds: DBCredentials, cb: (err: Error) => void): void;
+
+    /**
+     * Type representing the trace context object.
+     *
+     * @since 6.7.0
+     */
+    type TraceContext = Record<string, any>;
+
+    /**
+     * Base class for handling tracing.
+     *
+     * @since 6.7.0
+     */
+    class TraceHandlerBase {
+        constructor();
+
+        /**
+         * Checks if sending traces is enabled.
+         */
+        isEnabled(): boolean;
+
+        /**
+         * Enables sending traces.
+         */
+        enable(): void;
+
+        /**
+         * Disables sending traces.
+         */
+        disable(): void;
+
+        /**
+         * Called before invoking a public async method.
+         * @param traceContext  input/output trace context object.
+         */
+        onEnterFn(traceContext?: TraceContext): void;
+
+        /**
+         * Called after invoking a public async method.
+         * @param traceContext input/output trace context object.
+         */
+        onExitFn(traceContext?: TraceContext): void;
```

Comment:
https://node-oracledb.readthedocs.io/en/latest/api_manual/oracledb.html#onExitFn

The docs seem to indicate that the parameter is not optional

Reply:
Thanks. Fixed it.